monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/AvailablePlugins.tsx

```diff
@@ -33,10 +35,12 @@ const AvailablePlugins = (props) => {
   const authComponent = new AuthComponent({});
 
   useEffect(() => {
-    setDisplayedPlugins(availablePlugins);
-    setFilters((prevState) => {
-      return {...prevState, installed: filterInstalledPlugins};
-    });
+    if (availablePlugins !== null) {
+      setDisplayedPlugins(availablePlugins);
+      setFilters((prevState) => {
+        return {...prevState, installed: filterInstalledPlugins};
+      });
+    }
```

Comment:
Let's keep the logic as it was and use a different prop to indicate a connection error?